src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

```diff
@@ -17,16 +19,16 @@ export interface NameRequestDetailsIF {
 
 // Name request applicant details interface
 export interface NameRequestApplicantIF {
-  firstName: string
-  middleName: string
-  lastName: string
-  emailAddress: string
-  phoneNumber: string
-  addressLine1: string
-  addressLine2: string
-  addressLine3: string
-  city: string
-  countryTypeCode: string
-  postalCode: string
-  stateProvinceCode: string
+  firstName?: string
```

Comment:
It feels like this IF doesn't have much point if everything is optional. Would it make more sense to have the IF itself optional? (I'm not sure.)

Reply:
o i didn't know we could do that, i think that WOULD make more sense.

Reply:
Updated in next commit